ui/app/AppLayouts/Profile/views/CommunitiesView.qml

```diff
@@ -52,8 +99,17 @@ SettingsContentBase {
             }
 
             CommunitiesListPanel {
+                id: communitiesList
                 width: parent.width
-                model: root.profileSectionStore.communitiesList
+
+                model: SortFilterProxyModel {
+                    id: filteredModel
+
+                    sourceModel: root.profileSectionStore.communitiesList
+                    filters: [
+                        ExpressionFilter { expression: model.joined }
```

Comment:
Would better be a "ValueFilter", sth like:

```qml
filters: ValueFilter {
           roleName: "joined"
           value: true
       }
```

Reply:
Thank you, didn't know about it, fixed